### PR TITLE
py-torch-geometric: add v2.1.0

### PR DESCRIPTION
**Pull request by adamjstewart**
Successfully builds on macOS 12.6 (arm64) with Python 3.9.13 and Apple Clang 14.0.0.

@aweits @qwertos

### CODE REVIEW

**Review by tldahlgren — DISMISSED** [review]
Confirmed all sha256 based on change from url to `pypi`.

**Comment by tldahlgren** [comment]
Deferring merge to give at least one maintainer/requested reviewer a chance to comment.

File: var/spack/repos/builtin/packages/py-torch-geometric/package.py [hunk 1]
```diff
@@ -19,40 +19,54 @@ class PyTorchGeometric(PythonPackage):
     graphs as well as on 3D meshes or point clouds."""
 
     homepage = "https://github.com/rusty1s/pytorch_geometric"
```
Comment (nicholas-sly) [inline var/spack/repos/builtin/packages/py-torch-geometric/package.py]: Not strictly a part of your changes, but based on the pypi page, the homepage should probably be https://github.com/pyg-team/pytorch_geometric

File: var/spack/repos/builtin/packages/py-torch-geometric/package.py [hunk 2]
```diff
@@ -19,40 +19,54 @@ class PyTorchGeometric(PythonPackage):
...
     depends_on("python@3.6:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
-    depends_on("py-pytest-runner", type="build")
-    depends_on("py-torch", type=("build", "run"))
-    depends_on("py-numpy", type=("build", "run"))
     depends_on("py-tqdm", type=("build", "run"))
+    depends_on("py-numpy", type=("build", "run"))
     depends_on("py-scipy", type=("build", "run"))
-    depends_on("py-networkx", type=("build", "run"))
-    depends_on("py-scikit-learn", type=("build", "run"))
```
Comment (nicholas-sly) [inline var/spack/repos/builtin/packages/py-torch-geometric/package.py:37]: You seem to have removed py-scikit-learn and not added it anywhere else.  I assume this was intended?

**Review by tldahlgren — APPROVED** [review]

**Comment by nicholas-sly** [comment]
Looks good to me.

**Pull request merged**